packages/modelviewer.dev/examples/loading/index.html

```diff
@@ -411,7 +454,7 @@ <h4></h4>
       <div style="margin-top:24px;" class="copyright">©Copyright 2018-2020 Google Inc. Licensed under the Apache License 2.0.</div>
       <div id='footer-links'></div>
   </div>
-      
+
```

Comment:
Thanks for coffeemat.glb; where is it from? Can we show some kind of attribution/license? It should probably have a listing here in this footer.

Reply:
It's originally from Sketchfab, under CC Attribution license – good point on attribution! Also included on this three.js example: https://threejs.org/examples/?q=compress#webgl_loader_gltf_compressed

Will add attribution for the artist similarly 👍